### ISSUE TITLE
Add description

### ISSUE THREAD

**Issue by rbstrachan** [contributor-ninja/contributor.ninja#11]
Add a description to your project so people understand the goals of your project.

**Comment by xtuc** [comment]
Yes, I'll try to add a description in the README and maybe also on the website.

Unfortunately, I don't have much time to spend on contributor.ninja.

**Comment by ghost** [comment]
I would like to add description.
